Cartfile

```diff
@@ -1,15 +1,15 @@
 ### Internal
 
 github "kickstarter/Kickstarter-Prelude" "9118beee2b37fa5fc5a4f56502c1fa99030b9f9e"
-github "kickstarter/Kickstarter-ReactiveExtensions" "48a3f507995a6bdae964ddf9f2347af6bf74a19f"
+github "kickstarter/Kickstarter-ReactiveExtensions" "807153fe7e2cdb9ebcdc46862458e9e2c6b70d94"
```

Comment:
This SHA will be updated once https://github.com/kickstarter/Kickstarter-ReactiveExtensions/pull/82 merges.